Classes_Homework5/modules/file.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+class Files:
+    def __init__(self, target_file):
+        self.target_file = target_file
+
+    def get_path(self):
+        """Generates path to the target file (file name is configurable).
+        Will write file to current working directory"""
+        target_path = os.path.join(os.getcwd())+'\\'+self.target_file
```

Comment:
The **join** here takes only 1 parameter, so it doesn't work. The line should be:

```target_path = os.path.join(os.getcwd(), self.target_file)```

Reply:
End of review in scope of HW5.

Reply:
Thank you! But i'm afraid you are not reviewing the latest version, this piece of code was changed quite a while ago